alliance/config/settings.py

```diff
@@ -112,10 +145,9 @@
 )
 
 PROJECT_APPS = (
-    'alliance.core',
-    'alliance.apps.backlog',
-    'alliance.apps.accounts',
-    'apps.shared'
+    'apps.shared',
+    'apps.backlog',
+    'apps.accounts',
 )
```

Comment:
Ah ha - the change made to util.py, line 2, must be the key to making these paths work. I no longer get my import issues. Excellent!